docs/management/advanced-options.asciidoc

```diff
@@ -545,15 +545,6 @@ The maximum number of buckets a datasource can return. High numbers can have a n
 [[visualization-visualize-heatmapChartslibrary]]`visualization:visualize:legacyHeatmapChartsLibrary`::
 Disable this option if you prefer to use the new heatmap charts with improved performance, legend settings, and more..
 
-[float]
-[[kibana-telemetry-settings]]
-==== Usage Data
-
-[horizontal]
-[[telemetry-enabled-advanced-setting]]`telemetry:enabled`::
-When enabled, helps improve the Elastic Stack by providing usage statistics for
-basic features.
```

Comment:
This part is no-longer relevant because this setting only shows in the global settings